runtime/zephir.c

```diff
@@ -139,7 +139,7 @@ static void zephir_compile_program(zval *program TSRMLS_DC)
 
 	zend_hash_internal_pointer_reset_ex(ht, &pos);
 	for (
-	 ; zend_hash_get_current_data_ex(ht, (void**) &z, &pos) == SUCCESS
+	 ; z = zend_hash_get_current_data_ex(ht, &pos) != NULL
```

Comment:
[Why I change]
 
In PHP 7+, `zend_hash_get_current_data_ex` accept only 2 params and return value by `position`, check should != Null (null because C lang doesnot support nullptr and NULL is (void) *0)